awx/api/generics.py

```diff
@@ -765,6 +769,34 @@ def update(self, request, *args, **kwargs):
 
     def partial_update(self, request, *args, **kwargs):
         self.update_filter(request, *args, **kwargs)
+
+        if request.content_type == 'application/json-patch+json':
+            patch = JsonPatch(request.data)
+            try:
+                if not isinstance(request.data, list):
+                    raise JsonPatchException(
+                        "The patch must be supplied as a list",
+                    )
+                obj = self.get_object()
+                fields = [
+                    field for field in obj._meta.get_fields()
```

Comment:
This would have to properly handle secret decryption, given that one of the main use cases here is `credential.inputs`.

Reply:
If they're mostly interested in patching credential inputs, what are they really saving as opposed to doing a regular patch / put besides a few fields? :thinking:

Reply:
> If they're mostly interested in patching credential inputs

I think some of these settings are compelling, too, like the nested LDAP/SAML other auth dicts.